src/main/webapp/app/entities/modeling-exercise/modeling-exercise-update.component.html

```diff
@@ -1,9 +1,9 @@
 <form name="editForm markdown-preview" role="form" novalidate #editForm="ngForm">
-    <div class="modal-header">
-        <h4 class="modal-title" id="myModelingExerciseLabel" jhiTranslate="artemisApp.modelingExercise.home.createOrEditLabel">Create or edit a Modeling Exercise</h4>
-        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" (click)="clear()">&times;</button>
+    <div>
+        <h4 *ngIf="!modelingExercise.id" id="myModelingExerciseLabel" jhiTranslate="artemisApp.modelingExercise.home.createLabel">Create a new Modeling Exercise</h4>
```

Comment:
I try to use *ngIf; else as much as possible as it makes it obvious that there are 2 choices here (create/edit)

Reply:
Right, I will use it.